packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -3016,6 +3016,15 @@ void main() {
     expect(getBorderWeight(tester), 1.0);
   });
 
+  TextStyle _getLabelStyle(WidgetTester tester) {
```

Comment:
Could you move this function closer to where the other global functions are located in this file (the top), and also remove the `_`.